e2e/playwright-ci.config.js

```diff
@@ -4,12 +4,12 @@
 
 // eslint-disable-next-line no-unused-vars
 const { devices } = require('@playwright/test');
-const MAX_FAILURES = 5;
+const MAX_FAILURES = 0;
```

Comment:
this is just temporary as we address all @unstable tags

Reply:
@unlikelyzero does this need to stay in this PR?